front/src/gc-components/GCListItem.tsx

```diff
@@ -0,0 +1,10 @@
+import { List, MantineNumberSize } from "@mantine/core";
```

Comment:
If you are not using MantineNumberSize then delete it

Reply:
I deleted